test/jdk/javax/swing/regtesthelpers/JRobot.java

```diff
@@ -153,7 +168,7 @@ public void clickMouse(int buttons) {
      * Perform a click with the first mouse button.
      */
     public void clickMouse() {
-        clickMouse(InputEvent.BUTTON1_MASK);
+        clickMouse(InputEvent.BUTTON1_DOWN_MASK);
```

Comment:
Note that this is a different int value than the old MASK constant so it is important that you ran all tests (which you say you did) in case some test is not consistently using the Robot API and will end up mixing old and new which will likely cause failures.

Reply:
In this case presses and release happening inside clickMouse overloaded function, this will not mix old and new key     
 ```
public void clickMouse(int buttons) {
        mousePress(buttons);
        mouseRelease(buttons);
        delay();
    }
```